test/python/tensorflow/tf_unittest_ngraph.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/tensorflow/python/framework/test_util.py b/tensorflow/python/framework/test_util.py
+index aa8ecf8161..b8d2266132 100644
+--- a/tensorflow/python/framework/test_util.py
++++ b/tensorflow/python/framework/test_util.py
+@@ -971,6 +971,7 @@ class TensorFlowTestCase(googletest.TestCase):
+     self._threads = []
+     self._tempdir = None
+     self._cached_session = None
++    import ngraph
```

Comment:
Import `ngraph` is now nee to be changed to `ngraph_config`.

Reply:
@avijit-nervana : Made the change. Ready to merge now (I think)